src/shared/components/header/Header.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import type { Meta, StoryObj } from '@storybook/nextjs';
+import { User } from '@/shared/types/user';
+import Header from './Header';
+
+/**
+ * Header 컴포넌트 스토리
+ *
+ * - 전역 상단 헤더 컴포넌트
+ * - 로그인 상태에 따라 서로 다른 UI를 렌더링합니다.
+ * - 화면 크기에 따라 일부 크기가 변경됩니다.
+ *
+ * 상태 구분
+ * - 비로그인: 로그인 / 회원가입 버튼 노출합니다.
+ * - 로그인: 알림 영역 + 아바타 + 닉네임 노출합니다.
+ */
+
+const meta: Meta<typeof Header> = {
+  title: 'Shared/Header',
+  component: Header,
+  parameters: {
+    layout: 'fullscreen',
+  },
+  argTypes: {
+    isLoggedIn: {
+      control: 'boolean',
+      description: '로그인 여부 확인',
+    },
+    nickname: {
+      control: 'text',
+      description: '사용자 닉네임',
+    },
+    user: {
+      control: false,
+      description: '로그인한 사용자 정보',
+    },
+  },
+};
```

Comment:
<img width="1046" height="713" alt="Image" src="https://github.com/user-attachments/assets/c06afc66-d10e-4b4a-b5d3-8ea629fdd5b4" />

데코레이터 넣어주시면 fixed여서 스토리북 안보이는 문제 해결할 수 있어요!

```suggestion
const meta: Meta<typeof Header> = {
  title: 'Shared/Header',
  component: Header,
  parameters: {
    layout: 'fullscreen',
  },
  argTypes: {
    isLoggedIn: {
      control: 'boolean',
      description: '로그인 여부 확인',
    },
    nickname: {
      control: 'text',
      description: '사용자 닉네임',
    },
    user: {
      control: false,
      description: '로그인한 사용자 정보',
    },
  },
  decorators: [
    (Story) => (
      <div className='relative h-80 w-full'>
        <Story />
      </div>
    ),
  ],
};
```